modules/administration-guide/pages/show-deprecated-editors.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: PROCEDURE
+:description: Show deprecated editors
+:keywords: administration guide, show deprecated, dashboard, editors
+:navtitle:  Show deprecated editors
+:page-aliases: installation-guide:show-deprecated-editors.adoc
+
+[id="show-deprecated-editors"]
+= Show deprecated editors
+
+Learn how to show deprecated {prod-short} editors.
+
+NOTE: By default, the deprecated editors are hidden in the Dashboard UI. To show them, you need to set the `CHE_SHOW_DEPRECATED_EDITORS` environment variable to `true`.
```

Comment:
since the procedure describes how to show the deprecated editors, do we need this note?

I can move the information about deprecated editors being hidden to the introduction.